course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -28,22 +28,21 @@
      "language": "python",
      "metadata": {},
      "outputs": [],
-     "prompt_number": 1
+     "prompt_number": 24
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
       "## The matplotlib figure\n",
       "\n",
-      "At the heart of **every** matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen. Any object that can be drawn in this way is known as an \"Artist\" in matplotlib.\n",
```

Comment:
I think it is useful to introduce the `Artist` concept here.

Edit: looking down, I see you have entirely dropped the word "artist" from the notebook. I think this is a significant retrograde step as artists are the official terminology that matplotlib uses for [everything that can be drawn](http://matplotlib.org/api/artist_api.html?highlight=artist#module-matplotlib.artist).